app/views/workflows/_form.html.erb

```diff
@@ -14,7 +14,7 @@
 
   <%= f.text_field :name, required: true, id: 'name_field' %>
 
-  <%= f.select :batch_host, OODClusters.map { |key, val| [ "#{val.title} (#{val.resource_mgr_server.host})", key ] }, { label: "Batch Server:" }, { class: "selectpicker", id: "batch_host_select", required: true } %>
+  <%= f.select :batch_host, OODClusters.map { |cluster| [ "#{cluster.metadata_config[:title]} (#{cluster.job_config[:host]})", cluster.id ] }, { label: "Batch Server:" }, { class: "selectpicker", id: "batch_host_select", required: true } %>
```

Comment:
See above comment about not `using job_config` and changing the label from "Batch Server" to "Cluster"